javascript/javascript3/week3/script.js

```diff
@@ -0,0 +1,240 @@
+class Circle{
+    constructor(x,y,r,startAngle,endAngle,fillColor, ctx){
+        this.x = x;
+        this.y = y;
+        this.r = r;
+        this.startAngle = startAngle;
+        this.endAngle = endAngle;
```

Comment:
Since we're already breaking the contract described in the homework, the start and end angle of all circles are always the same! So maybe we don't even need to send this in, instead we could hard code those values in the draw method!